Avalonia.sln

```diff
@@ -38,7 +38,9 @@ EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Shared", "Shared", "{A689DEF5-D50F-4975-8B72-124C9EB54066}"
 	ProjectSection(SolutionItems) = preProject
 		.editorconfig = .editorconfig
+		src\Shared\EnumHelper.cs = src\Shared\EnumHelper.cs
```

Comment:
I have a last question before approving :)
Can this simply live in `Avalonia.Base` or it is used in projects not referencing that project?

Reply:
Yes. I've reverted it.